code/modules/mob/living/simple_animal/abnormality/teth/simple_smile.dm

```diff
@@ -74,6 +74,10 @@
 
 	src.pulled(pick(pullable))
 
+/mob/living/simple_animal/hostile/abnormality/smile/Initialize()
+  if(IsCombatMap())
+    density = FALSE
+
```

Comment:
```suggestion
/mob/living/simple_animal/hostile/abnormality/smile/Initialize()
  ..()
  if(IsCombatMap())
    density = FALSE

```
You need the ..() there else it breaks the abno.

Reply:
Thinking a bit more and this change would be better being moved to !abno_overwrites.dm

Reply:
Id rather keep it on smile rather than moving it to overwrites since this change is intended for both modes considering Smile was in LCL previously and has been used by admins for LCL and its also a buff directed at rca too